TSE REVIEW

carlos-ssh opened the pull request:
**Hi TSE Team I have ready this branch for merge. Please check it out and I'm able for changes and suggestions.
Thanks.**

[Live Demo Link](https://rawcdn.githack.com/carlos-ssh/tnw-page/23e93ca5f80139a758ac379fae0b0253e1017464/index.html)

[Original Page](https://thenextweb.com/)

_Happy Coding!_

**Carlos-SSH**

carlos-ssh committed:
we added the bootstrap stylesheet and scripts

carlos-ssh committed:
We worked with the media queries for diferent screens pending to solve some issues

AdesojiCodeMaster committed:
Added folders and files to the root.

AdesojiCodeMaster committed:
we reset the html file

AdesojiCodeMaster committed:
We modified the files and arranged the html structure.

AdesojiCodeMaster committed:
We added the remaining images and icons and fit them to html file.

AdesojiCodeMaster committed:
modified style.css and index.html files

AdesojiCodeMaster committed:
modified style.css and index.html files

carlos-ssh committed:
we finished the style of big screens and we gonna start to work with middle and small screens

carlos-ssh committed:
we are fixing issues in the style

AdesojiCodeMaster committed:
We styled and put images on html file

AdesojiCodeMaster committed:
We resolved the confilcts in the files

AdesojiCodeMaster committed:
Styled the images below the header part

carlos-ssh committed:
we fixed issues in the style

carlos-ssh committed:
working with the style

AdesojiCodeMaster committed:
Fixed the img files

AdesojiCodeMaster committed:
fixed the image file and aligned them

AdesojiCodeMaster committed:
Merge branch 'feature' of github.com:carlos-ssh/tnw-page into feature

AdesojiCodeMaster committed:
fixed the footer part

carlos-ssh committed:
fixing issues in the grid style

AdesojiCodeMaster committed:
Put the image in sizes and shape, correct overflow along x-axis

AdesojiCodeMaster committed:
set up responsiveness on html and css files

AdesojiCodeMaster committed:
Set responsiveness of the webpage for small/medium screen

AdesojiCodeMaster committed:
Increase the image responsiveness.

AdesojiCodeMaster committed:
fixed images on different screen sizes and fonts resizes

carlos-ssh committed:
we are working in some issues that we found in different screen sizes

AdesojiCodeMaster committed:
Adjusted the images

AdesojiCodeMaster committed:
Adjust the media query

AdesojiCodeMaster committed:
Reset the media query

AdesojiCodeMaster committed:
Reset the reponsiveness

AdesojiCodeMaster committed:
Adjusted the font

AdesojiCodeMaster committed:
Rest the date

AdesojiCodeMaster committed:
Aligned items on the page for responsiveness.

AdesojiCodeMaster committed:
Resizing the fonts

AdesojiCodeMaster committed:
font resizing

AdesojiCodeMaster committed:
Aligned items on the page for responsiveness.

carlos-ssh committed:
fixing some issues that we founded in the css file

AdesojiCodeMaster committed:
attached files to the project

AdesojiCodeMaster committed:
resolved conflicts

carlos-ssh committed:
fixed issues in margins and fixed the sticky menu in small screens

carlos-ssh committed:
fixed issues in margin lists

carlos-ssh committed:
fixed issues in spaces between words

carlos-ssh committed:
fixed issues positioning items in different screen sizes

carlos-ssh committed:
fixed issues in sizes images

carlos-ssh committed:
fixed issues in the viewport 1024px

AdesojiCodeMaster committed:
setting the outlook on different screen sizes

AdesojiCodeMaster committed:
code cleanup

AdesojiCodeMaster committed:
correct color format on css file

AdesojiCodeMaster committed:
code cleanup

AdesojiCodeMaster committed:
edited README.md file

carlos-ssh committed:
fixed issues in middle screens and adjusted grids

carlos-ssh committed:
fixed more issues in middle screen

carlos-ssh committed:
we finished the small screen issues

carlos-ssh committed:
we fixed issues in big screen

carlos-ssh committed:
upgraded readme file

carlos-ssh committed:
added some details at the readme file

carlos-ssh committed:
added the live demo link

carlos-ssh committed:
fixed some issues in and id duplicated we changed to class

carlos-ssh committed:
upgraded the live demo link to a new one version

carlos-ssh committed:
fixed issues in hover a-tag

carlos-ssh committed:
added the stickler.yml file

carlos-ssh committed:
fixed some issues in the main grid that it was affecting the responsivness

carlos-ssh committed:
fixed issues in big screen and cleaned the css file with linter

carlos-ssh committed:
fixed some issues in grids

carlos-ssh committed:
fixed issues with the linter in css file

carlos-ssh committed:
deleted node files from the repo

carlos-ssh committed:
updated the live demo link

carlos-ssh committed:
erasing ids from CSS that was affecting the functionality

carlos-ssh committed:
fixing issues in CSS file with linter code

carlos-ssh committed:
upgraded parameters in different files

AdesojiCodeMaster committed:
Realigned,rearranged all components for responsiveness

AdesojiCodeMaster committed:
clean up the code

AdesojiCodeMaster committed:
resolved conflicts

AdesojiCodeMaster committed:
realigned components for responsiveness

AdesojiCodeMaster committed:
resolved conflicts

AdesojiCodeMaster committed:
reset components

AdesojiCodeMaster committed:
reset components

AdesojiCodeMaster committed:
reset components

AdesojiCodeMaster committed:
reset components at the header part

AdesojiCodeMaster committed:
code cleanup

AdesojiCodeMaster committed:
realigned images

AdesojiCodeMaster committed:
Edited READme file.

AdesojiCodeMaster committed:
code cleaup

AdesojiCodeMaster committed:
code cleaup

carlos-ssh committed:
we cleaned the css file on lint

AdesojiCodeMaster committed:
code cleaup

carlos-ssh committed:
we are using the style.css file for quieries.

we are using the style.css file for quieries.

carlos-ssh committed:
Delete reset.css

AdesojiCodeMaster committed:
restructuring the file part

AdesojiCodeMaster committed:
code cleaup

AdesojiCodeMaster committed:
resolve the conflict

AdesojiCodeMaster committed:
restructuring the file part

AdesojiCodeMaster committed:
code clean-up

AdesojiCodeMaster committed:
realigned components

AdesojiCodeMaster committed:
Edited READMe file

carlos-ssh committed:
unnecessary files from the repo

carlos-ssh committed:
fixing responsive issues

carlos-ssh committed:
fixed issues in responsive screens

carlos-ssh committed:
css file cleaned with linter

AdesojiCodeMaster committed:
reset the project and cleanup the codes

AdesojiCodeMaster committed:
cleanup the codes

AdesojiCodeMaster committed:
cleanup the codes

AdesojiCodeMaster committed:
Edited READMe File

AdesojiCodeMaster committed:
Fixed the issues pointed out

AdesojiCodeMaster committed:
Fixed the issues pointed out

AdesojiCodeMaster committed:
aligned contents

AdesojiCodeMaster committed:
Fixed the iinter errors

AdesojiCodeMaster committed:
Put live link into READMe file

AdesojiCodeMaster committed:
realigned contents

AdesojiCodeMaster committed:
realigned more components

AdesojiCodeMaster committed:
Edited READMe file

AdesojiCodeMaster committed:
fixed issues pointed out

AdesojiCodeMaster committed:
Code cleanup

AdesojiCodeMaster committed:
fixed READMe file

AdesojiCodeMaster committed:
fixed the issues

AdesojiCodeMaster committed:
Edited READMe file

AdesojiCodeMaster committed:
fixed the issues

AdesojiCodeMaster committed:
fixed issues

AdesojiCodeMaster committed:
Aligned items

AdesojiCodeMaster committed:
fixed linter errors

AdesojiCodeMaster committed:
Edited READMe File

AdesojiCodeMaster committed:
fixed issues

AdesojiCodeMaster committed:
correcting issues

carlos-ssh committed:
code clean in html and css files fixed issues in responsive design

carlos-ssh committed:
fixed issues in the bio-tnw in middle screen and fixed issues in margins of founding nav

carlos-ssh committed:
fixed issues in founding h4 position

carlos-ssh committed:
fixed issues in the main grid

carlos-ssh committed:
fixed issues in spans and headers

carlos-ssh committed:
fixing issues in grids

carlos-ssh committed:
fixing issues in the styles bio-tnw big screen and added new fonts

carlos-ssh committed:
changing font families in the body

carlos-ssh committed:
fixing issues in responsive design

carlos-ssh committed:
fixing issues in the footer and biotnw container

carlos-ssh committed:
fixing issues in responsive design created a new css file

carlos-ssh committed:
fixed issues in the latest news grid

carlos-ssh committed:
fixed issues in funding container

carlos-ssh committed:
working on responsiveness grids

carlos-ssh committed:
fixing issues in the footer

carlos-ssh committed:
moved from address the css file to the root repo feature

carlos-ssh committed:
fixed issues in main grid and added font style to all elements

carlos-ssh committed:
html file cleaned from errors in anchors

carlos-ssh committed:
lintered the CSS file its cleaned now

carlos-ssh committed:
added a font folder because the url isnt working and fixed issues in anchor tags that was in blue and change the color

carlos-ssh committed:
fixed issues in the anchor tag color

carlos-ssh committed:
fixed issues in the anchor tags color

carlos-ssh committed:
fixed font color in the yellow nav

carlos-ssh committed:
fixed issues in the yellow navbar style

carlos-ssh committed:
fixed linter issues in css file

carlos-ssh committed:
fixed linter issues

carlos-ssh committed:
fixed requested issues

carlos-ssh committed:
fixed the main img size

carlos-ssh committed:
Fixing issues requested in some items

carlos-ssh committed:
cleaned html and css files with linter and validators

andresporras3423 commented on index.html:
```diff
@@ -1,11 +1,589 @@
+                                <h2 class="funding-header">Latest funding rounds on Index.co ></h2>
+                            </a>
+                            <ul class="funding-anchors">
+                                <li> <a href="#" style="color: #f42;"> See more</a></li>
+                                <li> <a href="#" style="color: #f42;"> Get updates by email </a></li>
+                                <li> <a href="#" style="color: #f42;"> Add info</a></li>
```
Don't use inline styles in your HTML.

andresporras3423 commented on style.css:
```diff
@@ -1,4 +1,795 @@
+  height: 45px;
+}
+
+.funding-grid-item4 {
+  display: none;
+}
```
This selector has the same style definition of the selector on line 133. Group selectors or create a new class.

andresporras3423 reviewed (changes requested):
### Status: Changes Required :recycle:
**Hello @AdesojiCodeMaster @carlos-ssh** 
Good job with your new submission for this project :muscle:  
Your page meets most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [x] Don't open a new pull request for each review.
- [x] Don't repeat yourself, in order to minimize code you can group selectors with the same style definition, or you can create a new class.
- [x]  Don't use inline styles in your HTML
- [ ] I think you need to update the live demo. In your CSS you have the correct breakpoint but your demo uses a different one (with 640px for small screen size).

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

carlos-ssh committed:
fixed requested issues in the repository

carlos-ssh committed:
fixed issues in a whitespace line css

carlos-ssh committed:
fixed issues in grids text

carlos-ssh committed:
fixed spaces in css file

carlos-ssh committed:
fixing spaces in css file

carlos-ssh committed:
fixing issues with linter

carlos-ssh committed:
fixed issues with linter in css file

carlos-ssh committed:
fixed issues in ul container

carlos-ssh commented:
**Hello TSE's we made the next changes:**

-  we don't go to create a new pull request for each review.
- we minimized code, we grouped selectors with the same style definition.
- Deleted inline styles in the HTML file.
- Fixed issues in 640px for small screen sizes.

[Live demo](https://rawcdn.githack.com/carlos-ssh/tnw-page/37b683b03bbfd623ae98bbf04075aed259ad8116/index.html#)

[Original](https://thenextweb.com)

Thanks.

Carlos.

tirthajyoti-ghosh reviewed (approved):
# Approved ✅

## Go ahead and merge it. Your project meets all the requirements.

Happy coding 👨‍💻

carlos-ssh commented:
Thanks for your review!.
